src/components/Avatar/Avatar.jsx

```diff
@@ -93,7 +97,7 @@ const Avatar = ({
     <div className={cx(AVATAR_CSS_BASE_CLASS, className)}>
       <div
         className={cx(
-          { 'remove-border': removeBorder },
+          { "without-border": withoutBorder },
```

Comment:
please use bem helper (pass an element paraeter as other and state parameter (without border))